README.md

```diff
@@ -6,6 +6,24 @@
 These models are used to communicate between MAPI and the native apps.
 This repository defines the protobuf [schema](./proto/collection.proto) for "blueprint" collections.
 
+## Validating Schema Changes
+
+We use [protolock](https://github.com/nilslice/protolock) to help us catch any unintentional breaking changes.
+In the proto directory a `proto.lock` file exists which captures a verbose state of the latest version of the proto file.
+When making changes to the proto file a [check](.github/workflows/version-compatibility.yml) will run on the PR to validate that [rules](https://github.com/nilslice/protolock) are enforced.
+The enforcement of rules helps to make sure that we don't accidentally introduce breaking changes.
+
+PRs will fail if there are any updates in the proto file that aren't recorded in the proto.lock. In order to update the lock file:
+- [Install](https://github.com/nilslice/protolock#install) protolock.
```

Comment:
Perhaps if this PR is merged I could create a setup script / makefile to automate the install of this tool (and the tool to perform [linting](https://github.com/guardian/mobile-apps-api-models/pull/22) too, and anything else contributors would need...)